src/DnsResolver/README.md

```diff
@@ -75,7 +75,7 @@ require:
 input-file:
   - $(this-folder)/resources/specification/dnsresolver/resource-manager/Microsoft.Network/preview/2020-04-01-preview/dnsresolver.json
```

Comment:
Now that swagger is published, there's no need to use a local swagger file to generate anymore. Here's an example of how to configure it in readme
```yaml
branch: 7a2cc29033fe4027ef421267f1684efbd0d40a93
input-file:
  - $(repo)/specification/web/resource-manager/Microsoft.Web/stable/2020-12-01/StaticSites.json
```

Reply:
Changed as discussed.